db/system_keyspace.cc

```diff
@@ -954,6 +954,20 @@ schema_ptr system_keyspace::discovery() {
     return schema;
 }
 
+schema_ptr system_keyspace::group0_kv_store() {
+    static thread_local auto schema = [] {
+        auto id = generate_legacy_id(NAME, GROUP0_KV_STORE);
+        return schema_builder(NAME, GROUP0_KV_STORE, id)
+            .with_column("key", utf8_type, column_kind::partition_key)
+            .with_column("value", utf8_type)
+            .with_version(generate_schema_version(id))
+            .set_wait_for_sync_to_commitlog(true)
+            .with_null_sharder()
+            .build();
+    }();
+    return schema;
```

Comment:
Why not allow any schema with CREATE TABLE?

Reply:
Reducing the scope of the first PR. Using a hardcoded table was the fastest way to let us focus on implementing the semantics of DML statements.

Of course one of the follow-up tasks will be to implement create keyspace/table for group 0 / strongly consistent tables.

Reply:
You'll have to specify that you want a group 0 table in the DDL, so it's an additional syntax.
We'll also have to choose things like the replication strategy in the code.

So handling group 0 DDLs is some work.

Reply:
I think this _is_ a new replication strategy.

Reply:
Probably yes. I don't want implementing a whole new replication strategy to be part of the first PR. Let's make it one of the follow-up tasks.

Reply:
As suggested by @kbr-, it will be changed in the future